src/lib/store.js

```diff
@@ -26,10 +26,13 @@ export function configureStore(appConfig) {
     error: {},
   };
 
+
+
   // configure the initial state
   const preloadedQueryState = appConfig.urlHandlerApi
     ? appConfig.urlHandlerApi.get(initialQueryState)
     : initialQueryState;
+  preloadedQueryState.hidden_params = appConfig.searchApi.axiosConfig.hidden_params
```

Comment:
While the idea was to keep the `state` as clean as possible and to store all user choices/input only, I like the simplicity of your solution. 👏 
My fear, and this is what we need to keep an eye on, is that we don't abuse of this. If we see the need to even add more things similar to this one, then we need to find a better solution.

I would really stress on remembering (maybe create an issue) on removing this when `axios` gets fixed, and probably add a comment here and/or in the state below to explain why.

Reply:
I would say that even after the `axios` issue gets fixed, it would still be good to store the hidden params in the state, i.e. making this not just a way to configure the search API client, but also a capability of RSK in general.

Also, another idea for this feature is to add this fully to the state since you might want to change these hidden params even after the app's initialization, e.g. you have in the UI tab views for "pending", "accepted", "rejected" and when the user clicks them, you change the hidden params state (`status=pending`), trigger a reload of the results, but don't update the browser URL with different `f=...` params.